server/data/_hidden/exams-list.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Can you `git rm` both this and the image list file from the PR?  They should be ignored by the gitignore, but depending on when this branch was created they could have still slipped through